### PR TITLE
Don't add quotes if the file name already has quotes

### PR DESCRIPTION
**Pull request by blasten**
cc @tvolkert

### CODE REVIEW

File: lib/src/interface/common.dart [hunk 1]
```diff
@@ -26,7 +26,7 @@ String sanitizeExecutablePath(String executable,
   if (!platform.isWindows) {
     return executable;
   }
-  if (executable.contains(' ') && !executable.startsWith('"')) {
+  if (executable.contains(' ') && !executable.contains('"')) {
```
Comment (tvolkert) [inline lib/src/interface/common.dart:29]: What if it contains `\ ` (an escaped space)?

Comment (blasten) [reply lib/src/interface/common.dart:29]: I could be wrong, but my understanding is that:` C:\"foo"\bar` resolves as `C:\foo\bar`.  If this is true, then ` C:\"foo^ bar"` resolves as  ` C:\foo^ bar`.  (`^ ` is Windows way to escape spaces). I don't have a Windows computer, but let me ask to see if we can verify it.

**Review by tvolkert — APPROVED** [review]
LGTM.

I added you to the repo and pub publishers, so you should be able to merge and publish

File: lib/src/interface/common.dart (hunk 1 above)
Comment (blasten) [reply lib/src/interface/common.dart:29]: These paths are resolved correctly:
```.dart
Process.runSync('"C:\\Users\\franciscojma\\Hello World\\git.exe"', const <String>[]);
Process.runSync('\"C:\\Users\\franciscojma\\Hello World\\git.exe\"', const <String>[]);
```

The ones below returned "Process not found":
```.dart
Process.runSync('C:\\Users\\franciscojma\\"Hello World"\\git.exe', const <String>[]);
Process.runSync('C:\\Users\\franciscojma\\"Hello^ World"\\git.exe', const <String>[]);
Process.runSync('"C:\\Users\\franciscojma\\Hello^ World"\\git.exe"', const <String>[]);
Process.runSync('C:\\Users\\franciscojma\\Hello^ World\\git.exe', const <String>[]);
```

**Pull request merged**